python-sdk/nuscenes/prediction/tests/test_predict_helper.py

```diff
@@ -12,12 +12,16 @@
 
 
 class MockNuScenes(NuScenes):
-    """ Mocks the NuScenes API needed to test PredictHelper. """
 
     def __init__(self,
                  sample_annotations: List[Dict[str, Any]],
                  samples: List[Dict[str, Any]]):
-
+        """
+        Mocks the NuScenes API needed to test PredictHelper.
+        Note that we are skipping the call to the super class constructor on purpose to avoid loading the tables.
```

Comment:
Is it a proper practice?

Reply:
No, it's a hack that avoids breaking backward-compatibility.